.github/workflows/main.yml

```diff
@@ -161,6 +161,10 @@ jobs:
       - name: doxygen check
         run: |
           make -C doc
+      - name: installation check
+        run: |
+          make install
+          ls -lR $GITHUB_WORKSPACE/test-install/include $GITHUB_WORKSPACE/test-install/bin $GITHUB_WORKSPACE/test-install/lib
```

Comment:
As `share` is missing here I suggest simply `ls -lR $GITHUB_WORKSPACE/test-install`

Reply:
I did this originally, but the doxygen files went on forever.  If you are happy with that, I can easily make this change.  I wanted the ability to be able to do a visual check, as well as error out if there were issues.

Reply:
Have gone with your suggested change and code pushed.